src/ucai/core/databricks.py

```diff
@@ -197,6 +258,7 @@ def create_function(
         # return self.client.functions.create(function_info)
         raise ValueError("sql_function_body must be provided.")
 
+    @retry_on_session_expiration()
```

Comment:
Do we need `()`? Can we use @retry_on_session_expiration

Reply:
It was needed for the factory approach, but after removing that additional layer that supports arg passing, it's no longer needed. Removed!